### ISSUE TITLE
autofocus should be removed from signup form

### ISSUE THREAD

**Issue by ndunn219** [pennersr/django-allauth#2476]
In the following code:

```
class BaseSignupForm(_base_signup_form_class()):
    username = forms.CharField(label=_("Username"),
                               min_length=app_settings.USERNAME_MIN_LENGTH,
                               widget=forms.TextInput(
                                   attrs={'placeholder':
                                          _('Username'),
                                          'autofocus': 'autofocus'}))
```
`'autofocus': 'autofocus' `should be removed as noted in the warning at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefautofocus. Plus, it autofocuses on the username field instead of the email field, which comes first, for signup forms that include both.

**Comment by 9mido** [comment]
You can just use https://pypi.org/project/django-widget-tweaks/ to temporarily solve this.

**Comment by pennersr** [comment]
Closed via b9ca281c

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/2476
